app/routes/admin/content/edit/$id.tsx

```diff
@@ -0,0 +1,399 @@
+        defaultValue={content.author}
+        name={"author"}
+      />
+
+      <Input
+        type="date"
+        error={actionData?.errors?.createdAt}
+        label={"Opprettet (mm-dd-yyyy)"}
+        htmlRef={createdAtRef}
+        defaultValue={content.createdAt.split("T")[0]}
+        name={"createdAt"}
+      />
+
+      <div className="mt-3 grid grid-cols-2 pb-4 md:grid-cols-3">
+        <Toggle
+          leftText={"Nei"}
+          rightText={"Ja"}
+          label={"Fremhevet"}
+          ref={suggestedRef}
+          inputName={"suggested"}
+          defaultChecked={content.suggested}
+        />
+
+        <fieldset className="sm:mt-4 md:col-span-2">
+          <legend>Kategori:</legend>
+          <div className="mt-3 gap-4 md:flex">
+            {categories?.map((category) => {
+              return (
+                <div key={category.id} className="flex items-center gap-1">
+                  <input
+                    className="h-4 w-4 cursor-pointer"
+                    type="radio"
+                    ref={categoriesRef}
+                    name="categoryId"
+                    id="category"
+                    value={category.id}
+                    defaultChecked={content.categoryId === category.id}
+                  />
+                  <label className="font-bold" htmlFor="category">
+                    {category.text}
+                  </label>
+                </div>
+              );
+            })}
+          </div>
+
+          {actionData?.errors?.categoryId && (
+            <div className="pb-1 text-variant-pink-2" id="error">
+              {actionData?.errors?.categoryId}
+            </div>
+          )}
+        </fieldset>
+      </div>
+
+      <fieldset className="mt-3 grid grid-cols-2 gap-4 sm:grid-cols-5">
+        <legend>Tags:</legend>
+        {tags?.map((tag) => {
+          return (
+            <div key={tag.id} className="flex items-center gap-1 ">
+              <input
+                className="h-4 w-4 cursor-pointer"
+                type="checkbox"
+                ref={tagsRef}
+                name="tag"
+                id={tag.id}
+                value={tag.id}
+                defaultChecked={
+                  content.tags.findIndex((t) => t.id === tag.id) !== -1
+                    ? true
+                    : false
+                }
+              />
+              <label className="font-bold" htmlFor="tag">
+                {tag.text}
+              </label>
+            </div>
+          );
+        })}
+      </fieldset>
+
+      <div className="mt-5 flex justify-end">
+        <PrimaryButton
+          text={
+            transition.state === "submitting" || transition.state === "loading"
+              ? "Lagrer ... "
+              : "Lagre"
+          }
+          disabled={
+            transition.state === "submitting" || transition.state === "loading"
+          }
+        />
+      </div>
+    </Form>
+  );
+};
+
+export default EditContent;
```

Comment:
Det mangler ErrorBoundary flere plasser, men jeg og Mathilde kan ta det i en egen PR 🥳

Reply:
Yes, do it! :)

Reply:
Lagde #49